### DISCUSSION TITLE
Missing button to manually approve linked abilities?

### DISCUSSION THREAD

**Discussion by dowrite** [mitre/caldera#2710] [Q&A]
I'm doing the "Manual Operation" flag, but found that I'm missing the "Review Command" button. Not sure if I have a bad install or if this is a bug in Caldera version 4.1.0
![Screenshot_2022-12-28_16-28-45](https://user-images.githubusercontent.com/78618480/209849456-ed1c4939-f550-49c2-be84-69a78400961d.png)

**Comment by mkultraWasHere** [comment]
Addressing in #2729. Apologies for the delay.
